app/src/pages/EducadoAdmin.tsx

```diff
@@ -1,89 +1,91 @@
 import { useState } from "react";
 import useSWR from "swr";
 import { useNavigate, Link, useLocation } from 'react-router-dom';
-
+import {BACKEND_URL} from "../helpers/environment"
 // Services
 import AuthServices from "../services/auth.services";
 
-// Interfaces
-import { CCApp } from '../interfaces/CCApp'
-
 // Components
 import Loading from "../components/general/Loading";
 import Layout from "../components/Layout";
 import { PageDescriptor } from "../components/PageDescriptor";
 
-import {BACKEND_URL} from "../helpers/environment"
+import { getUserToken } from '../helpers/userInfo';
 
 
 
 const EducadoAdmin = () => {
+    //Variable to detect and determine the search term used to filter applications
     const [searchTerm, setSearchTerm] = useState('')
-
-    // Used to navigate to the detailed view for the single applicant
+    
+    //Location and navigation 
     let navigate = useNavigate();
     let location = useLocation();
+    const token = getUserToken();
 
+    //Get data from the relevant route
     const { data, error } = useSWR(
-        `${BACKEND_URL}/api/applications?approved=false`,
+        'api/applications',
```

Comment:
This URL is still given. It may not be wrong, but I don't get why. Can you explain?